NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/controller/EmployeeProfileController.java

```diff
@@ -2,26 +2,21 @@
 
 import com.nullterminators.project.model.EmployeeProfile;
 import com.nullterminators.project.service.EmployeeProfileService;
-import java.util.List;
-import java.util.Optional;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.DeleteMapping;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Why did you replace with .\*? Checkstyle does not support .\* imports

Reply:
Corrected it!